src/shapes/Text/Text.ts

```diff
@@ -941,18 +937,14 @@ export class Text<
    * @param {GraphemeBBox} graphemeInfo current grapheme box information
    * @param {Object} startingPoint position of the point
    */
-  _setGraphemeOnPath(
-    positionInPath: number,
-    graphemeInfo: GraphemeBBox,
-    startingPoint: XY
-  ) {
+  _setGraphemeOnPath(positionInPath: number, graphemeInfo: GraphemeBBox) {
     const centerPosition = positionInPath + graphemeInfo.kernedWidth / 2,
       path = this.path!;
 
     // we are at currentPositionOnPath. we want to know what point on the path is.
     const info = getPointOnPath(path.path, centerPosition, path.segmentsInfo)!;
-    graphemeInfo.renderLeft = info.x - startingPoint.x;
-    graphemeInfo.renderTop = info.y - startingPoint.y;
+    graphemeInfo.renderLeft = info.x - path.pathOffset.x;
+    graphemeInfo.renderTop = info.y - path.pathOffset.y;
```

Comment:
this change is not 1:1.
before we were subtracting the first path point all the time, while now we are just substracting the offset?

Reply:
because of the bug startingPoint was 0,0 if I am not mistaken (at least for paths starting with M cmd)

Reply:
shouldn't it have been undefined/undefined?

Reply:
ok i couldn't read this: `tempInfo = <TPathSegmentInfoCommon<'M'>>basicInfo;`

Reply:
i need a color scheme where the TS syntax is rgba (255, 255, 255, 0.02) fontSize: 1

Reply:
Explain